suite-native/module-home/src/components/PortfolioGraph.tsx

```diff
@@ -61,11 +59,11 @@ export const PortfolioGraph = () => {
         <>
             <PortfolioGraphHeader />
             {error ? (
-                <Box style={applyStyle(graphWrapperStyle)}>
-                    <Text variant="label" color="gray600">
-                        There are some troubles with loading graph points: {error}
-                    </Text>
-                </Box>
+                <GraphError
+                    error={error}
+                    onTryAgain={handleSelectTimeFrame}
+                    selectedTimeFrame={selectedTimeFrame}
```

Comment:
You don't need to pass this to error component, just make callback from content of useEffect above, and call it in that useEffect as wall as onTryAgain click

Reply:
should be done, thanks!